sys/shell/commands/sc_netstats_nb.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) Koen Zandberg <koen@bergzand.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for displaying neighbor statistics
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ * @author      Benjamin Valentin <benpicco@beuth-hochschule.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "net/gnrc/netif.h"
+#include "net/netstats.h"
+#include "net/netstats/neighbor.h"
+
+static void _print_neighbors(netif_t *dev)
+{
+    netstats_nb_t *stats = &dev->neighbors.pstats[0];
+    unsigned header_len = 0;
+    char l2addr_str[3 * L2UTIL_ADDR_MAX_LEN];
+    puts("Neighbor link layer stats:");
+    header_len += printf("L2 address               fresh");
+#ifdef MODULE_NETSTATS_NEIGHBOR_ETX
```

Comment:
Here and below:

```suggestion
#if IS_USED(MODULE_NETSTATS_NEIGHBOR_ETX)
```

At least for the headers

```suggestion
    if (IS_USED(MODULE_NETSTATS_NEIGHBOR_ETX)) {
```

could also work, however, since you have field access in the table content below (and thus need `#if IS_USED(…)`), I would prefer a unified scheme.